src/app/features/reporter/reporter-dialog.component.html

```diff
@@ -42,29 +39,134 @@
+		<div class="col-md-12 mt-4 mb-4">
+			<h4 class="mb-4">Upload Results</h4>
+			<table class="table table-bordered results-table">
+				<tbody>
+					<tr>
+						<td class="font-weight-bold">Total</td>
+						<td>{{totalTestsRun}}</td>
+					</tr>
+					<tr>
+						<td class="font-weight-bold">Passed</td>
+						<td>{{testsRun['passed']}}</td>
+					</tr>
+					<tr>
+						<td class="font-weight-bold">Failed</td>
+						<td>{{testsRun['failed']}}</td>
+					</tr>
+					<tr>
+						<td class="font-weight-bold">Not updated</td>
+						<td>{{testsRun['NotUpdated']}}</td>
+					</tr>
+					<tr>
+						<td class="font-weight-bold">Not existed</td>
+						<td>{{testsRun['NotExistsJamaInFiles']}}</td>
+					</tr>
+					<tr>
+						<td class="font-weight-bold">Test suite not in Jama</td>
+						<td>{{testsRun['FileNotInJama']}}</td>
+					</tr>
+				</tbody>
+			</table>
+		</div>
+		<div class="col-md-12 mt-4" *ngIf="areResultsWrong()">
+			<h4>Wrong uploads</h4>
+			<div *ngIf="testsWrong['failed'].length > 0">
+				<h5 class="text-secondary">Failed</h5>
+				<table class="table table-sm table-striped">
+					<thead>
+						<tr>
+							<th scope="col">Test</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr *ngFor="let test of testsWrong['failed']">
+							<td>{{test.name}}</td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
+			<div *ngIf="testsWrong['NotUpdated'].length > 0">
+				<h5 class="text-secondary">Not Updated</h5>
+				<table class="table table-sm table-striped">
+					<thead>
+						<tr>
+							<th scope="col">Test</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr *ngFor="let test of testsWrong['NotUpdated']">
+							<td>{{test.name}}</td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
+			<div *ngIf="testsWrong['NotExistsJamaInFiles'].length > 0">
+				<h5 class="text-secondary">Jama test not exists</h5>
+				<table class="table table-sm table-striped">
+					<thead>
+						<tr>
+							<th scope="col">Test</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr *ngFor="let test of testsWrong['NotExistsJamaInFiles']">
+							<td>{{test.name}}</td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
+			<div *ngIf="testsWrong['FileNotInJama'].length > 0">
+				<h5 class="text-secondary">Test Suite not exist in Jama</h5>
+				<table class="table table-sm table-striped">
+					<thead>
+						<tr>
+							<th scope="col">Test</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr *ngFor="let test of testsWrong['FileNotInJama']">
+							<td>{{test.name}}</td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
+		</div>
+	</div>
 </div>
 <systelab-dialog-bottom>
 	<button type="button" class="btn mr-0" (click)="doUpdateTestCase()">Update Test Case</button>
+	<div *ngIf="totalTestsRun > 0" class="progress allure-progress-bar" style="height: 20px;">
+		<div class="progress-bar bg-success" role="progressbar" [ngStyle]="{'width':testsRunPercentage + '%'}"
```

Comment:
As discussed, change the bg-success by bg-info for usability purposes in both progress bars